src/Workspaces/Core/Portable/PublicAPI.Shipped.txt

```diff
@@ -205,6 +205,15 @@ const Microsoft.CodeAnalysis.Classification.ClassificationTypeNames.XmlLiteralEn
 const Microsoft.CodeAnalysis.Classification.ClassificationTypeNames.XmlLiteralName = "xml literal - name" -> string
 const Microsoft.CodeAnalysis.Classification.ClassificationTypeNames.XmlLiteralProcessingInstruction = "xml literal - processing instruction" -> string
 const Microsoft.CodeAnalysis.Classification.ClassificationTypeNames.XmlLiteralText = "xml literal - text" -> string
+const Microsoft.CodeAnalysis.Classification.ClassificationTypeNames.FieldName = "field name" -> string
+const Microsoft.CodeAnalysis.Classification.ClassificationTypeNames.EnumFieldName = "enum field name" -> string
+const Microsoft.CodeAnalysis.Classification.ClassificationTypeNames.ConstantName = "constant name" -> string
+const Microsoft.CodeAnalysis.Classification.ClassificationTypeNames.LocalName = "local name" -> string
+const Microsoft.CodeAnalysis.Classification.ClassificationTypeNames.ParameterName = "parameter name" -> string
+const Microsoft.CodeAnalysis.Classification.ClassificationTypeNames.MethodName = "method name" -> string
+const Microsoft.CodeAnalysis.Classification.ClassificationTypeNames.ExtensionMethodName = "extension method name" -> string
```

Comment:
Any idea how we would expand this for extension everything?

Reply:
No idea what you mean with that - I can need the classification type names that get added so I made them public.